docs/services/robot-service.md

```diff
@@ -8,8 +8,11 @@ weight: 10
 tags: ["robot state", "services"]
 # SME: Cheuk
 ---
-Robot Service constitutes a minimal set of APIs that most robots (Viam Server, [Viam Python SDK](https://python.viam.dev/) and various SDKs) should support.
-Users will likely use the Robot Service as an entrypoint to interact with Viam robots and provide a way to get updates from the robot as a whole.
+
```

Comment:
should this still be in draft mode? and if not we need an icon for this page :)

Reply:
Yeah it should still be in draft mode; see comment below. I'd added a newline to it and temporarily un-drafted it when I was first working on the PR, and then forgot to remove it from the PR, so Sierra reviewed it and made some suggestions that I implemented but I didn't mean to actually look over this page at all in this PR.